README.md

```diff
@@ -42,5 +42,13 @@ let res = device.write(&buf).unwrap();
 println!("Wrote: {:?} byte(s)", res);
 ```
 
+# Cross-compiling
```

Comment:
Not sure if its a great idea to include documentation for cross compiling, maybe cut it down to just mentioning the relevant feature flags of the crate that help making it easier?

Reply:
Sure, I can do that.